ceno-client/test/erroringlcs.go

```diff
@@ -0,0 +1,38 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+)
+
+type ErrorCode int
+
+const INTERNAL_ERROR ErrorCode = 2140
+const ERR_MSG string = "Testing that we can serve errors and have the client auto-refresh erro pages"
+
+type ErrorResult struct {
+	ErrCode  ErrorCode
+	ErrMsg   string
+	Complete bool
+	Found    bool
+	Bundle   string
```

Comment:
JSON error responses do not include "complete", "found" or "bundle" in the specification documents. Could this create an issue? Integration seems to be fine.